### PR TITLE
Sample App to demonstrate Authentication

### PR DESCRIPTION
**Pull request by amareshsm**
A sample code of Freshchat that demonstrates consumption of it's own Freshchat APIs.

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
#230

File: Freshworks-Samples/Freshchat/authentication-demonstration/app/dialog.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                <th>Email</th>
+                <th>Role</th>
+                <th>Groups</th>
+            </tr>
+        </tbody>
+    </table>
+    <script src="dialog.js"></script>
+</body>
+
+</html>
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshchat/authentication-demonstration/app/dialog.html:28]: Add a new empty line at the end of file in all the files.

File: Freshworks-Samples/Freshchat/authentication-demonstration/app/template.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<html>
+
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshchat/authentication-demonstration/app/template.html:4]: Please remove jQuery from all the HTML files and replace jQuery functions with Vanilla JS.

File: Freshworks-Samples/Freshchat/authentication-demonstration/app/app.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    .then((data) => {
+                        let response = JSON.parse(data.response);
+                        //on success 
+                        listAllAgents(response);
+                    }).catch((error) => {
+                        //on failure 
+                        showError(error);
+                    });
+                //shows currently logged-in agent.
+                client.data.get('loggedInAgent')
```
[the remark this answers is not in the document]
Comment (amareshsm) [reply Freshworks-Samples/Freshchat/authentication-demonstration/app/app.js:70]: In this app I'm listing all the agents so only I just displayed currently logged-in agent.

**Comment by amareshsm** [comment]
I hope all are corrections done if any thing needs to be fixed please let me know

**Review by Saif-Shines — APPROVED** [review]

**Pull request merged**